packages/invoices/src/OutboundShipment/DetailView/modals/BatchesTable.tsx

```diff
@@ -1,64 +1,32 @@
 import React from 'react';
```

Comment:
In general not much has changed in here, it's still just being passed the lines to render and a callback for editing the quantity. The status is returned from a hook now